_includes/footer.html

```diff
@@ -8,37 +8,43 @@ <h1 class="add-shadow">Latest Posts</h1>
     <div class="download">
       <h1 class="add-shadow align-left">Download App</h1>
       <p class="white-text align-left">Get opsdroid today</p>
-      <p class="align-left"><a class="align-left" href="https://github.com/opsdroid/opsdroid-desktop" title="Opsdroid Desktop App" target="_blank">
-        <i class="fas fa-download fa-lg"></i> Opsdroid Desktop
-      </a></p>
+      <p class="align-left"><a class="align-left" href="https://github.com/opsdroid/opsdroid-desktop"
+          title="Opsdroid Desktop App" target="_blank">
+          <i class="fas fa-download fa-lg"></i> Opsdroid Desktop
+        </a></p>
     </div>
 
     <div class="follow">
       <h1 class="add-shadow align-left">Follow Us</h1>
       <p class="white-text align-left">Don't miss a thing</p>
       <div class="align-left social-icons">
-          <a href="https://github.com/{{ site.github_username }}" title="Opsdroid Github Repo"><i class="fab fa-github fa-lg"></i></a>
-          <a href="https://gitter.im/opsdroid" title="Opsdroid Gitter Channel"><i class="fab fa-gitter fa-lg"></i></a>
-          <a href="https://twitter.com/{{ site.twitter_username }}" title="Opsdroid Twitter Account"><i class="fab fa-twitter fa-lg"></i></a>
-          <a href="https://medium.com/opsdroid" title="Opsdroid Medium Blog"><i class="fab fa-medium-m fa-lg"></i></a>
-          <a href="https://www.youtube.com/channel/UC0FeGkDF7FrshmzAsb9lDvw" title="Opsdroid Youtube Account"><i class="fas fa-caret-right fa-lg"></i></a>
+        <a href="https://github.com/{{ site.github_username }}" title="Opsdroid Github Repo"><i
+            class="fab fa-github fa-lg"></i></a>
+        <a href="https://riot.im/app/#/room/#opsdroid-general:matrix.org" title="Opsdroid Gitter Channel"><i
```

Comment:
Text also needs changing here

```suggestion
        <a href="https://riot.im/app/#/room/#opsdroid-general:matrix.org" title="Opsdroid Matrix Channel"><i
```